src/experimental-layers/src/path-marker-layer/create-path-markers.js

```diff
@@ -72,9 +74,20 @@ function createMarkerAlongPath({path, percentage, lineLength, color, object, vie
     .multiply(new Vector2(along, along))
     .add(path[i]);
 
-  const vDirection2 = new Vector2(viewport.projectFlat(path[i + 1]))
-    .subtract(viewport.projectFlat(path[i]));
+  const projectFlat = coordinateSystem === COORDINATE_SYSTEM.METER_OFFSETS ?
+   obj => {
+    const [dx,dy] = viewport.metersToLngLatDelta(obj);
+    const [x,y] = coordinateOrigin;
+    console.log(obj, [x,y], [dx,dy]);
+    return viewport.projectFlat([x-dx, dy+y]); // <<< TODO signs
+  } : obj => console.log(obj) || viewport.projectFlat(obj);
```

Comment:
We have a `log` class if you want to log. No raw `console` calls please.

Reply:
Sorry, wasn't going to commit them.